### PR TITLE
fix operator file naming

### LINKED ISSUE #5658: Fix typo in operator file name
**Task description**:
Fix the typo(`mainfests.go` --> `manifests.go`) in the following file name:
- https://github.com/karmada-io/karmada/blob/master/operator/pkg/controlplane/apiserver/mainfests.go
- https://github.com/karmada-io/karmada/blob/master/operator/pkg/controlplane/etcd/mainfests.go
- https://github.com/karmada-io/karmada/blob/master/operator/pkg/controlplane/metricsadapter/mainfests.go
- https://github.com/karmada-io/karmada/blob/master/operator/pkg/controlplane/search/mainfests.go
- https://github.com/karmada-io/karmada/blob/master/operator/pkg/controlplane/webhook/mainfests.go

**Note: This task should wait for #5536 to avoid conflicts.**

**Solution**:

**Who can join or take the task**:

The good first issue is intended for `first-time contributors` to get started on his/her contributor journey.

After a contributor has successfully completed 1-2 good first issue's,
they should be ready to move on to `help wanted` items, saving the remaining `good first issue` for other new contributors.

**How to join or take the task**:

Just reply on the issue with the message `/assign` in a separate line.

Then, the issue will be assigned to you.

**How to ask for help**:

If you need help or have questions, please feel free to ask on this issue.
The issue author or other members of the community will guide you through the contribution process.

### PR DESCRIPTION
**Pull request by husnialhamdani**
**What type of PR is this?**

**What this PR does / why we need it**:
Fix typo in operator file name

**Which issue(s) this PR fixes**:
Fixes #5658 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: operator/pkg/controlplane/apiserver/mainfests
Comment (RainbowMango) [inline operator/pkg/controlplane/apiserver/mainfests:1]: The target file name should be `manifests.go`.  What we are going to fix is the typo, not removing the suffix.

PS: I'm sorry I missed `.go` in the previous issue.

Comment (husnialhamdani) [reply operator/pkg/controlplane/apiserver/mainfests:1]: updated

**Comment by RainbowMango** [comment]
Hi @husnialhamdani Thanks for the quick response.
You might need to 
1. Sign the DCO according to [the guideline](https://github.com/karmada-io/karmada/pull/5740/checks?check_run_id=32002392248) to make CI happy.
2. Squash the commits according to [the guideline](https://karmada.io/docs/contributor/github-workflow/#squash-commits).

Then we are ready to move forward.

**Comment by husnialhamdani** [comment]
> Hi @husnialhamdani Thanks for the quick response. You might need to
> 
> 1. Sign the DCO according to [the guideline](https://github.com/karmada-io/karmada/pull/5740/checks?check_run_id=32002392248) to make CI happy.
> 2. Squash the commits according to [the guideline](https://karmada.io/docs/contributor/github-workflow/#squash-commits).
> 
> Then we are ready to move forward.

Hi @RainbowMango,
Could you please check?

**Comment by RainbowMango** [comment]
This PR now contains a commit that it shouldn't have.

**Review by RainbowMango — APPROVED** [review]
Hi @husnialhamdani I helped to squash the commits as I see you might have some trouble with it.

/lgtm
/approve

**Pull request merged**